sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -348,14 +348,14 @@ def test_radius_neighbors_classifier_outlier_labeling():
     # Test radius-based classifier when no neighbors found and outliers
     # are labeled.
 
-    X = np.array([[1.0, 1.0], [2.0, 2.0]])
-    y = np.array([1, 2])
+    X = np.array([[1.0, 1.0], [2.0, 2.0], [0.99,0.99], [0.98, 0.98], [2.01,2.01]])
```

Comment:
I've modified it again, I don't understand why it's coming back each time... Thank you.